src/frontend/components/DashboardVideoLiveControlPane/index.tsx

```diff
@@ -14,28 +19,93 @@ import { DashboardVideoLiveWidgetVisibilityAndInteraction } from './widgets/Dash
 import { DashboardVideoLiveWidgetVOD } from './widgets/DashboardVideoLiveWidgetVOD';
 import { DashboardVideoLiveWidgetSharedLiveMedia } from './widgets/DashboardVideoLiveWidgetSharedLiveMedia';
 
+const messages = defineMessages({
+  titleConfiguration: {
+    defaultMessage: 'configuration',
+    description:
+      'Title of the tab used to configure the live in capital letters',
+    id: 'components.DashboardVideoLiveControlPane.titleConfiguration',
+  },
+  titleAttendance: {
+    defaultMessage: 'attendances',
+    description:
+      'Title of the tab used to watch attendance of the live in capital letters',
+    id: 'components.DashboardVideoLiveControlPane.titleAttendance',
+  },
+});
 interface DashboardVideoLiveControlPaneProps {
   video: Video;
 }
-
 export const DashboardVideoLiveControlPane = ({
   video,
 }: DashboardVideoLiveControlPaneProps) => {
+  const intl = useIntl();
+  const extendedTheme: ThemeType = {
+    ...theme,
+    tab: {
+      active: {
+        color: 'blue-focus',
+      },
+      color: 'blue-active',
+      hover: {
+        color: 'blue-focus',
+      },
+      extend:
+        " font-family: 'Roboto-Bold';\
+        font-weight: bold;\
```

Comment:
As the font is bold there is no need to add again the font-weight. On firefox it's really bolder than chrome.